src/subgraph/dev/endpoints.ts

```diff
@@ -1,10 +1,10 @@
 // subgraph URLs
 
 // Goerli
-const GOERLI_SG_URL_DEV = 'https://api.studio.thegraph.com/query/51391/thebadge-goerli-dev/1.0.6'
+const GOERLI_SG_URL_DEV = 'https://api.studio.thegraph.com/query/51391/thebadge-goerli-dev/version/latest'
```

Comment:
why should we put latest version again in? in my opinion we should handle always an specific version because of possible breaking changes

Reply:
mmm that might make sense in staging or production but not on development, we want to quickly deploy the subgraph for testing without having to make a new release of the SDK, I will leave dev in latest and the rest in fixed versions

Reply:
Ahhh okay I got it